class.jetpack.php

```diff
@@ -5956,14 +5956,16 @@ public static function dns_prefetch( $new_urls = null ) {
 	public function wp_dashboard_setup() {
 		if ( self::is_active() ) {
 			add_action( 'jetpack_dashboard_widget', array( __CLASS__, 'dashboard_widget_footer' ), 999 );
+			$widget_title = __( "Jetpack", 'jetpack' );
 		} elseif ( ! self::is_development_mode() && current_user_can( 'jetpack_connect' ) ) {
 			add_action( 'jetpack_dashboard_widget', array( $this, 'dashboard_widget_connect_to_wpcom' ) );
+			$widget_title = __( "Please Connect Jetpack", 'jetpack' );;
```

Comment:
Fixed in https://github.com/Automattic/jetpack/commit/56e77efc2c40e3f5436d70af80cbb2f276f581c1